client/js/organizationReservations.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+/*
+ * organizationReservations.js
+ * Copyright(c) 2016 Universidad de Las Palmas de Gran Canaria
+ * Authors:
+ *   Jaisiel Santana <jaisiel@gmail.com>
+ *   Alejandro Sánchez <alemagox@gmail.com>
+ *   Pablo Fernández <pablo.fernandez@ulpgc.es>
+ * MIT Licensed
+
+*/
+var connectionsAPI;
+var utils;
+var clientLogic;
+
+//initialization
+var initReservations = function() {
+
+  $('#popWindow').modal();
+
+  clientLogic.setUpDrawModule();
+
+  //get franchise from url
+  var franchise = window.location.search.replace('?', '');
```

Comment:
I think we need to find another way to detect what is the current franchise rather than rely on the URL